CONTRIBUTING.md

```diff
@@ -137,26 +130,3 @@ function f () {
   return x;
 }
 ```
-
-- Separate imports into 2 blocks: absolute imports and relative imports.
-
-```diff
-import { a } from 'react';
-+
-import { b } from '../my/moduleB';
-```
-
-- Sort imports alphabetically by package name. The character '@' comes before alphanumerical characters.
```

Comment:
removed all these rules from the guidelines, since now they are enforced by the linter